### PR TITLE
Added UT for 3D Layout and Hfss Field Plots on Nets and Layers.

### PR DESCRIPTION
**Pull request by maxcapodi78**
Working from 23 R2

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM

File: _unittest/test_41_3dlayout_modeler.py [hunk 1]
```diff
@@ -729,8 +749,8 @@ def test_92_import_edb(self):
 
     @pytest.mark.skipif(config["desktopVersion"] < "2022.2", reason="Not Working on Version earlier than 2022R2.")
```
Comment (PipKat) [inline _unittest/test_41_3dlayout_modeler.py:750]: Even in tests, messages should be short, complete sentences. Version for customer-facing doc should always be in this format: 2022R2. Consider rewriting the reason as this: This test does not work on versions earlier than 2022 R2.

Comment (maxcapodi78) [reply _unittest/test_41_3dlayout_modeler.py:750]: I can change it but I didn't touch that UT and that comment in this PR

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Pull request merged**